### PR TITLE
Follow up comments on sig scalability charter

### PR DESCRIPTION
**Pull request by wojtek-t**
/assign @spiffxp

### CODE REVIEW

File: sig-scalability/block_merges.md [hunk 1]
```diff
@@ -7,8 +7,11 @@ from merging into the relevant repos. This document describes the underlying
 ### Rules of engagement.
 The rules of engagement for blocking merges are as following:
 
-- Observe as scalability regression on one of release-blocking test suites.
-- Block merges of all PRs.
+- Observe as scalability regression on one of release-blocking test suites
```
[the remark this answers is not in the document]
Comment (wojtek-t) [reply sig-scalability/block_merges.md]: Correct.

> (we don't appear to have a stable3 equivalent, should we?)

Apparently - I wasn't aware of that.

@spiffxp - I don't want to make this part of this PR. If at some point we decide to change the set (by we I mean sig-scalability+sig-release (maybe +sig-testing) representatives), it shouldn't require changing this doc.
Ideally, it should link to the place where those are listed, but I can't really find this link.

File: sig-scalability/block_merges.md [hunk 2]
```diff
@@ -7,8 +7,11 @@ from merging into the relevant repos. This document describes the underlying
...
+  (defined as green to red transition - if tests were already failing, we
+  don't have a right to declare a regression).
+- Block merges of all PRs to the relevant repos, declaring which repos those
```
[the remark this answers is not in the document]
Comment (wojtek-t) [reply sig-scalability/block_merges.md]: yes - it's all about a single branch. Clarified.

**Comment by wojtek-t** [comment]
PTAL

File: sig-scalability/block_merges.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (spiffxp) [reply sig-scalability/block_merges.md]: Agree, I was just highlighting to make sure we're ok with the discrepancies.  eg: we're not doing 5K node performance regression tests against patch releases, and the lack of any coverage on stable3 (I'll open an issue to correct the latter)

IMO we should find a way to make it clear which tests you're talking about though, the tests are all named inconsistently.  "jobs owned by sig scalability that live on the release-master-blocking dashboard" is close but requires jumping through a number of files to connect.  I'll open an issue suggesting we break those jobs into their own file(s) in test-infra as a start.

**Comment by spiffxp** [comment]
Opened https://github.com/kubernetes/test-infra/issues/9282 and https://github.com/kubernetes/test-infra/issues/9283 as followup

**Pull request merged**